packages/vats/src/provisionPool.js

```diff
@@ -0,0 +1,159 @@
+// @jessie-check
+// @ts-check
+
+import { fit, M, makeScalarMapStore } from '@agoric/store';
+import { E, Far } from '@endo/far';
+// TODO: move to narrower package?
+import { observeIteration, observeNotifier } from '@agoric/notifier';
+import { AmountMath, AmountShape } from '@agoric/ertp';
+import { WalletName } from '@agoric/internal';
+import { makeMyAddressNameAdmin, PowerFlags } from './core/basic-behaviors.js';
+
+const { details: X, quote: q } = assert;
+
+const privateShape = harden({
+  poolBank: M.eref(M.remotable('bank')),
+});
+
+/**
+ * Given attenuated access to the funding purse,
+ * handle requests to provision smart wallets.
+ *
+ * @param {(address: string, depositFacet: ERef<DepositFacet>) => Promise<void>} sendInitialPayment
+ */
+const makeBridgeProvisionTool = sendInitialPayment => {
+  /**
+   * @param {{
+   *   bankManager: ERef<BankManager>,
+   *   namesByAddressAdmin: ERef<NameAdmin>,
+   *   walletFactory: ERef<import('@agoric/vats/src/core/startWalletFactory').WalletFactoryStartResult['creatorFacet']>
+   * }} io
+   */
+  const makeHandler = ({ bankManager, namesByAddressAdmin, walletFactory }) =>
+    Far('provisioningHandler', {
+      fromBridge: async (_srcID, obj) => {
```

Comment:
`obj` doesn't say enough about what this param is for.

Reply:
It's a payload coming over the bridge. Is `payload` significantly better?

This name is conventional. `ck fromBridge|grep .js|grep obj|wc -l` shows 15 occurrences.
Consistency seems reasonably valuable.

Reply:
> Is payload significantly better?

It seems so to me.